packages/framework/src/Resources/scripts/admin/plugins/jquery.counterup.js

```diff
@@ -28,7 +28,9 @@
             var num = $this.attr('data-value');
             var nums = [num];
             var isComma = /[0-9]+,[0-9]+/.test(num);
+            var isSpace = /\s/.test(num);
             num = num.replace(/,/g, '');
+            num = num.replace(/\s/g, '');
```

Comment:
can you do it on line 32 ? or is this because of diff because the external library ???

Reply:
Yes I do not want to change previous code